src/bin/cratesfyi.rs

```diff
@@ -3,14 +3,14 @@ use std::fmt::Write;
 use std::path::PathBuf;
 use std::sync::Arc;
 
+use anyhow::{anyhow, Context as ErrContext, Error, Result};
```

Comment:
`Context` isn't named outside of this import, right?
```suggestion
use anyhow::{anyhow, Context as _, Error, Result};
```